app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -33,6 +38,10 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
   // ref allows us to avoid a memory leak
   const mountedRef = useRef(false);
 
+  // this is the place to start and should be the only place to set up the locationsUrl
```

Comment:
this is a suspicious comment to me, who is the audience here? I'd also expect us to either use the constants below, or have some sort of "read from context" step?